app/config/smsConfigsLoader.js

```diff
@@ -3,7 +3,7 @@
  */
 app.ConfigName = {
   CAMPAIGN_TRANSITIONS: 'start_campaign_transition',
-  DONORSCHOOSE: 'donorschoose',
+  DONORSCHOOSE: 'donorschoose_moco_campaigns',
```

Comment:
It's very confusing that this was working at all.. but I'm still unclear on what's going on in this code